apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -657,6 +657,8 @@ suite('navigation >', function() {
 
       window.Navigation = NavigationFactory(fakeWindow);
       navigator.mozHasPendingMessage.withArgs('notification').returns(false);
+
+      Navigation.setReady();
```

Comment:
yes I think it should be OK. If you label it correctly, maybe with a comment to this bug, it won't be confusing. Remember unit tests is a tool to prevent old bugs from coming back